x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ssl/StoreKeyConfigTests.java

```diff
@@ -52,6 +53,42 @@ public void testKeyStorePathCanBeEmptyForPkcs11() throws Exception {
         assertThat(ee.getCause().getMessage(), containsString("PKCS11 not found"));
     }
 
+    public void testCreateKeyManagerFromPKCS12ContainingCA() throws Exception {
+        assumeFalse("Can't run in a FIPS JVM", inFipsJvm());
+        final Settings settings = Settings.builder().put("path.home", createTempDir()).build();
+        final String path = getDataPath("/org/elasticsearch/xpack/security/transport/ssl/certs/simple/httpCa.p12").toString();
```

Comment:
I'd add load the keystore once here and assert that it has two aliases, and then use their names to do `keyManager.getPrivateKey()` below, so that we are certain that the keymanager filtered out the aliases it should